README.md

```diff
@@ -709,6 +709,81 @@ Accept: application/json
 | *response.expectedCompletionTimestamp* | no        | The UNIX time stamp at which the Data Subject Request is expected to be completed |
 | *response.requestID*                   | no        | The request ID known to the destination system                                    |
 
+## Consent
+
+A Consent Request is initiated when a [Data Subject](#Subject) specifies or changes consent preferences.
+
+### Consent Request
+
+```http request
+POST /endpoint HTTP/1.1
+Host: www.example.com
+Content-Type: application/json
+Accept: application/json
+Authorization: Bearer $auth
+
+{
+  "apiVersion": "dsr/v1",
+  "kind": "ConsentRequest",
+  "metadata": {
+    "uid": "22880925-aac5-42f9-a653-cb6921d361ff",
+    "tenant": "axonic"
+  },
+  "request": {
+    "controller": "axonic",
+    "property": "axonic.io",
+    "environment": "production",
+    "regulation": "gdpr",
+    "jurisdiction": "eugdpr",
+    "identities": [
+      {
+        "identitySpace": "account_id",
+        "identityFormat": "raw",
+        "identityValue": "123"
+      }
+    ],
+    "purposes": {
+      "advertising": true,
+      "data_sales": true,
+      "email_mktg": false
+    },
```

Comment:
Do we need to stringify the consent value so that the falsey values appear? 

Also not sure if legal basis useful. Current shoreline ```"behavioral_advertising_vlabs":{"allowed":"false","legalBasisCode":"consent_optin"}```